Include/Apple/Protocol/AppleVoiceOver.h

```diff
@@ -48,13 +48,33 @@ typedef enum {
   AppleVoiceOverAudioFileUsernameOrPasswordIncorrect = 0x05, ///< UsernameOrPasswordIncorrect
   AppleVoiceOverAudioFileAccountLockedTryLater       = 0x06, ///< AccountLockedTryLater (*)
   AppleVoiceOverAudioFileAccountLocked               = 0x07, ///< AccountLocked (*)
+  AppleVoiceOverAudioFileIndexNonLocalised           = 0x3B, ///< Files from here have no localised version
   AppleVoiceOverAudioFileVoiceOverBoot               = 0x3B, ///< VoiceOver_Boot (*)
   AppleVoiceOverAudioFileVoiceOverBoot2              = 0x3C, ///< VoiceOver_Boot (*)
   AppleVoiceOverAudioFileClick                       = 0x3D, ///< Click (*)
   AppleVoiceOverAudioFileBeep                        = 0x3E, ///< Beep
   AppleVoiceOverAudioFileMax                         = 0x3F,
 } APPLE_VOICE_OVER_AUDIO_FILE;
 
+/**
+  Is localised?
+**/
+#define AppleVoiceOverAudioFileIsLocalised(File)  ((File) < AppleVoiceOverAudioFileIndexNonLocalised)
```

Comment:
Better to stick to CAPS for all macros still (e.g. EFI_ERROR).

Reply:
Changed in upcoming commit.